pkg/authenticator/authenticator.go

```diff
@@ -63,46 +67,30 @@ func NewAuthenticator(logger *zap.Logger) (Authenticator, string, error) {
 		return authenticator, credentialType, nil
 	}
 
-	logger.Error("IBMCLOUD_CREDENTIALS_FILE undefined, trying to read storage secret store")
-	conf, err := config.ReadConfig(logger)
+	// Parse it the secret is storage-secret-store
```

Comment:
Can we have default case as well in switch case at line 64 and return the error

Reply:
This is taken care of in the parseIBMCloudCredentials function, where AUTH_TYPE is validated. If there is any other AUTH_TYPE, that function will return error